CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file. This projec
 ### Deprecated
 - The `element` parameter on `backup:list` is deprecated. Use the `--element=` option instead. (#1563)
 - The `wait()` function of `Workflows` is deprecated. Use `checkStatus()` instead. (#1584)
+- The `User::getOrgMemberships()` is deprecated. Use `User::getOrganizationMemberships()` instead. (#1613)
```

Comment:
This comment is not specific to `User::getOrganizationMemberships()`, but somewhere in the Terminus docs -- maybe the CHANGELOG might be a good place -- there should be a recommendation to put `"compatible-version": "^1.0.1"` in their plugin's composer.json file, as any plugin that uses APIs introduced in 1.0.1 will not work with Terminus 1.0.0.